vacasa/connect/connect.py

```diff
@@ -1846,3 +1846,62 @@ def update_unit_reservation_buffer(self, unit_reservation_buffer_id: int, params
 
     return self._patch(url, json={'data': {
         'type': 'unit-reservation-buffer', 'attributes': params}}, headers=headers).json()
+
+def get_unit_blocks(self, params: dict = None):
+    """
+    get a list of unit blocks
+    """
+
+    url = f"{self.endpoint}/v1/unit-blocks"
+    headers = self._headers()
+
+    return self._iterate_pages(url, headers, params)
+
+def get_unit_unit_block_by_id(self, unit_block_id: int):
+    """
+    get an unit block by id
+    """
+
+    url = f"{self.endpoint}/v1/unit-blocks/{unit_block_id}"
+    headers = self._headers()
+
+    return self._get(url, headers).json()
+
+def create_unit_block(self,
+                                   unit_id: int,
+                                   unit_reservation_buffer_id: int,
+                                   unit_block_type_id: int,
+                                   days_out: int,
+                                   note: str):
+    """
+    create a unit block
+    """
```

Comment:
With this many arguments it would be helpful to include them in the docstring. We typically use the Google format of Python docstrings which look something like this:
```python
    """Brief description of my function

    Args:
        param1: The first parameter.
        param2: The second parameter.

    Returns:
        The return value
    """
```

Reply:
On it, thank you!

Reply:
Done!